railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_6_0.rb.tt

```diff
@@ -8,3 +8,10 @@
 
 # Don't force requests from old versions of IE to be UTF-8 encoded
 # Rails.application.config.action_view.default_enforce_utf8 = false
+#
+# Purpose and Expiry information are embbedded inside signed and encrypted
+# cookies for increased security.
+#
+# This option is not backwards compatible with earlier Rails versions.
+# It's best enabled when your entire app is migrated and stable on 6.0.
```

Comment:
I was just about to ask you to cut down the past documentation that was here and make it easier to understand. Then you pushed a version that did that! Whatever had you arrive at this wording and style, keep doing that!